src/filter.js

```diff
@@ -73,33 +74,22 @@ define(function (require) {
    *  }
    *  </code></div>
    */
-  p5.Filter = function(type) {
-    this.ac = p5sound.audiocontext;
 
-    this.input = this.ac.createGain();
-    this.output = this.ac.createGain();
 
-    /**
-     *  The p5.Filter is built with a
-     *  <a href="http://www.w3.org/TR/webaudio/#BiquadFilterNode">
-     *  Web Audio BiquadFilter Node</a>.
-     *  
-     *  @property biquadFilter
-     *  @type {Object}  Web Audio Delay Node
-     */
+
+  p5.Filter = function (type) {
+	p5.Effect.call(this);
     this.biquad = this.ac.createBiquadFilter();
```

Comment:
check the spacing on line 81 (but really I should add linting to do this automatically)